test/BUILD.bazel

```diff
@@ -0,0 +1,32 @@
+load("//:bats.bzl", "bats_test")
+
+bats_test(
+    name = "bats_test_suite",
+    srcs = ["bats.bats"],
+    deps = [
```

Comment:
That looks quote involved and I wonder why the internal .bash files need to be added. I'd eyxpect this to be part of the "install" script?

Anyways, I think we should have a dedicated test file to avoid failing tests due to missing dependencies, e.g. when new fixtures are added.

Reply:
Normally, this would all be replaced with `glob(["**"])` which would capture everything.

Unfortunately, this can't be done because bazel's "glob" function will report an error if it globs a file containing whitespace in the filename.  I have worked around it by "exploding" the glob so that glob is only used for the directories that don't contain a file with whitespace in the name.

Reply:
I've modified this rule to be a little easier to maintain.